Mathlib/LinearAlgebra/FiniteDimensional.lean

```diff
@@ -879,6 +879,10 @@ noncomputable def divisionRingOfFiniteDimensional (F K : Type*) [Field F] [Ring
   qsmul := _
 #align division_ring_of_finite_dimensional divisionRingOfFiniteDimensional
 
+lemma FiniteDimensional.isUnit (F : Type*) {K : Type*} [Field F] [Ring K] [IsDomain K]
+    [Algebra F K] [FiniteDimensional F K] {x : K} (H : x ≠ 0) : IsUnit x :=
+  letI := divisionRingOfFiniteDimensional F K; H.isUnit
```

Comment:
Why `letI` and not just `let`? (As far as I know, in Lean 4, the `I` is no longer related to instances, but now means "inline".)

Reply:
```suggestion
  let := divisionRingOfFiniteDimensional F K; H.isUnit
```